controllers/resourcePerson.js

```diff
@@ -46,9 +51,14 @@ exports.getSingleResourcePerson=(async(req,res,next)=>{
     try{
         const result =await ResourcePerson.getResourcePersonbyId(id);
         const results=result.recordset 
+        if(result.recordsets.length > 0){
         return res.status(200).json({
         msg: "Workshop details successfully fetched!",
```

Comment:
why is the message workshop details successfully fetched? we are fetching resource persons, right?